tasks/build.py

```diff
@@ -203,35 +204,50 @@ def build_jekyll(ctx, branch, owner, repository, site_prefix,
 
 
 @task
-def download_hugo(ctx, version='0.48'):
+def download_hugo(ctx):
+    HUGO_VERSION_PATH = CLONE_DIR_PATH / HUGO_VERSION
+    if HUGO_VERSION_PATH.is_file():
+        LOGGER.info(f'.hugo-version found')
+        hugo_version = ''
+        with HUGO_VERSION_PATH.open() as hugo_vers_file:
+            hugo_version = hugo_vers_file.readline().strip()
+            # escape-quote the value in case there's anything weird
+            # in the .hugo-version file
+            hugo_version = shlex.quote(hugo_version)
```

Comment:
The safety of shelx.quote() is shell-dependent, so the safest approach is for this content is to return an error if hugo_version doesn't match a regular expression of "only numerics and the '.' character".